src/EnergyPlus/DataGlobalConstants.cc

```diff
@@ -652,6 +658,146 @@ namespace DataGlobalConstants {
         return ResourceTypeChar;
     }
 
+    void ValidateFuelType(std::string FuelTypeInput)
+    {
+        // FUNCTION INFORMATION:
+        //       AUTHOR         Dareum Nam
+        //       DATE WRITTEN   May 2020
+        //       MODIFIED       na
+        //       RE-ENGINEERED  na
+
+        // PURPOSE OF THIS FUNCTION:
+        // Validates fuel types and sets output strings
+
+        auto const SELECT_CASE_var(FuelTypeInput);
```

Comment:
Would a switch statement be more efficient for these functions?

Reply:
Can't switch on a string, unfortunately.  In the functions that convert integer to the appropriate string, you can switch on the integers.

Reply:
Thanks for your insights. Since C++ can't switch on a string, once I used enum to convert strings to integer and make one standard function instead of the three functions.